### ISSUE TITLE
BUG: Cloned Token Rotates after being created

### ISSUE THREAD

**Issue by GambetTV** [grandseiken/foundryvtt-multilevel-tokens#77]
Kind of a weird one, but when you move your token into a Token Cloning area, the Cloned Token that is generated immediately turns in a different direction. Sometimes it turns 180 degrees, sometimes 90, sometimes like, 5. Once the token moves further it is fine, but it's just that first moment.

**Comment by grandseiken** [comment]
I haven't seen something like you described, and I can't reproduce this problem, do you have any other information or steps to show it?

**Comment by GambetTV** [comment]
Yeah, this is really weird, but I've tracked it down. It happens when you have this module, Monk's Token Bar, and Auto-Rotate all active at the same time. Disable either one of the others and it doesn't happen anymore. I have no idea why it would be associated with Monk's Token Bar, since to my knowledge that module doesn't affect the actual token at all, so it's pretty weird that it's a necessary component of duplicating this bug.
